functions.php

```diff
@@ -64,8 +64,8 @@ function betterChildBreadcrumbs() {
 		$childBreadcrumb = $startLink.$pageLink.$endLink.$pageTitle.$closeLink;
 	}
 
-	if ( $parentBreadcrumb != '' ) {echo '<span>'.$parentBreadcrumb.'</span>';}
-	if ( $childBreadcrumb != '' ) {echo '<span>'.$pageTitle.'</span>';}
+	if ( '' !== $parentBreadcrumb ) {echo '<span>'.$parentBreadcrumb.'</span>';}
```

Comment:
I can't run $parentBreadcrumb through one of the expected `esc_*()` functions because we've built it to have HTML tags. There's an alternative approach that might get past this, but it would mean a bit of re-engineering.